src/Components/Chakra__Tag.res

```diff
@@ -0,0 +1,796 @@
+open Chakra__MakeProps
+
+@react.component @module("@chakra-ui/react")
+external make: (
+  ~children: React.element=?,
+  /**
+    Component Props {Tag}
+   **/
```

Comment:
Please, add props type definition for `Tag`, You can see [**Here**](https://chakra-ui.com/docs/data-display/tag#props)
![image](https://user-images.githubusercontent.com/16365952/127855436-3e37075d-44d1-4b86-80d9-dc4c7fbdc83b.png)

You just need to add `variant` and `size` props type definition. 

Ask me if you need to help, thank you @fncolon

Reply:
look like variant is using badge variant, https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/tag.ts

solved by 1e78e8a617b8b8044495801927489e5cf2380344 commit